Assets/Scripts/PickUps.cs

```diff
@@ -36,33 +39,38 @@ private void OnTriggerEnter2D(Collider2D other)
         if (other.tag == "Player")
         {
             PlayerController player = other.GetComponent<PlayerController>();
+            
+            if (gameObject.name == "HealthUp(Clone)")
+            {
+                if (PlayerController.currentHP < player.maxHP)
+                {
+                    PlayerController.currentHP++;
+                    Instantiate(feedbacks, transform.position, transform.rotation);
+                    SoundManager.instance.PlayAcquire();
+                    Destroy(gameObject);
+                }
+            }
+
             if (gameObject.name == "SpeedUp(Clone)")
             {
-                if (player.speed < 7)
+                if (player.speed < speedLimite)
                 {
                     player.speed += .8f;
+                    Instantiate(feedbacks, transform.position, transform.rotation);
                 }
-                Destroy(gameObject);
             }
             
             if (gameObject.name == "ExtraBomb(Clone)")
             {
-                if (PlayerController.maxBomb < 5)
+                if (PlayerController.maxBomb < bombLimit)
                 {
                     PlayerController.maxBomb += 1;

```

Comment:
PlayerController.maxBomb += 1 could be turned into PlayerController.maxBomb++, correct?